pages/common/yank.md

```diff
@@ -0,0 +1,23 @@
+# yank
+
+> Read input from stdin and display a selection interface that allows a field to be selected and copied to the clipboard.
+
+- Yank an environment variable key or value:
```

Comment:
Start with the simplest command invocation - `yank`. Explain what it does and give an example.